src/Pyz/Zed/ProductManagement/ProductManagementDependencyProvider.php

```diff
@@ -148,7 +147,6 @@ protected function getProductAbstractEditViewExpanderPlugins(): array
     {
         return [
             new ScheduledPriceProductAbstractEditViewExpanderPlugin(),
-            new ProductApprovalProductAbstractEditViewExpanderPlugin(),
```

Comment:
We need to clarify it, if it's not needed anymore - we need to deprecate it first and remove it from all demoshops.